cloud/aws/terraform/misp/definition.tf

```diff
@@ -140,20 +142,40 @@ locals {
       name      = "misp-proxy"
       essential = true
       portMappings = [{
-        containerPort = 8080
-        hostPort      = 8080
+        containerPort = local.misp_proxy_port
+        hostPort      = local.misp_proxy_port
         protocol      = "tcp"
       }]
       environment = [{
         name  = "NGINX_PLACEHOLDER_HOSTNAME"
         value = local.placeholder_hostname
-      }, {
+        }, {
         name  = "NGINX_PORT"
         value = "8080"
-      }, {
+        }, {
         name  = "NGINX_PROXY_PASS"
         value = "http://localhost:80"
       }]
+    },
+    {
+      image     = "linuxserver/openssh-server:latest"
+      name      = "ssh-server"
+      essential = true
+      portMappings = [{
+        containerPort = 2222
+        hostPort      = 2222
+        protocol      = "tcp"
+      }]
+      environment = [{
+        name  = "PUBLIC_KEY"
+        value = "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAINBtjQQ3SxIzW5IhviByGdr3Y0x5vroJRf6Rw9DM7jD7"
```

Comment:
Would it be possible to create the key-pair ad-hoc before the instance is created?

Reply:
Is it possible that the version of the code that you commented isn't the latest one? Because the current behavior is that a keypair is created when the stack is deployed.